tests/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ option(OQS_ENABLE_TEST_CONSTANT_TIME "Build test suite with support for Valgrind
 if(CMAKE_C_COMPILER_ID STREQUAL "GNU" OR
    CMAKE_C_COMPILER_ID MATCHES "Clang")
     add_compile_options(-Wbad-function-cast)
+    add_compile_options(-g)
+    add_compile_options(-O0)
```

Comment:
Why did you add these flags?

Reply:
It was debugging purposes. Once I get circleCI tests passing, I will remove them. The 0 optimization flag was to prevent the compiler from optimizing many things out. Allowed easier debugging